doc/quickstart.md

```diff
@@ -4,10 +4,19 @@
 
 1. A supported SRIOV hardware on the cluster nodes. Currently the supported models are:
   
-   | Vendor-ID   | Device-ID   |
-   | ----------- | ----------- |
-   | Intel       | 158b        |
-   | Melanox     | 1015, 1017  |
+| Model                    | Vendor ID | Device ID |
+| ------------------------ | --------- | --------- |
+| Intel XXV710 25GbE SFP28 |  0x8086 | 0x158b |
+| Intel E810-CQDA2/2CQDA2 100 GbE dual-port QSFP28 | 8086  | 1592 |
+| Intel E810-XXVDA4 25 GbE quad-port SFP28 | 8086  | 1593 |
+| Intel E810-XXVDA2 25 GbE dual-port SFP28 | 8086  | 159b |
+| Mellanox MT27710 Family [ConnectX-4 Lx] 25GbE dual-port SFP28 | 0x15b3 | 0x1015 |
```

Comment:
So, for Mellanox NICs i prefer we dont specify the exact model as the support list may be long.
the ETH/IB/VPI , single/dual port, sfp/qsfp may get unbearably long.

Here is what we currently have for Mellanox in `api/v1/helper.go`:
```
	"15b3 1015 1016", // ConnectX-4LX
	"15b3 1017 1018", // ConnectX-5, PCIe 3.0
	"15b3 101b 101c", // ConnectX-6
	"15b3 1013 1014", // ConnectX-4
	"15b3 a2d6 101e", // MT42822 BlueField-2 integrated ConnectX-6 Dx
```

So this would translate for Mellanox NICs:

| Model | Vendor ID | Device ID |
| ------- | ----------- | ----------- |
| Mellanox MT27700 Family [ConnectX-4] | 15b3 | 1013 |
| Mellanox MT27710 Family [ConnectX-4 Lx] | 15b3 | 1015 |
| Mellanox MT27800 Family [ConnectX-5] | 15b3 | 1017 |
| Mellanox MT28908 Family [ConnectX-6] | 15b3 | 101b |
| Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx| 15b3 | a2d6 |

Reply:
Done! can you review again? Thanks.